src/wobjectimpl.h

```diff
@@ -696,7 +696,7 @@ static void registerMethodArgumentType(int _id, void **_a) {
         constexpr auto f = binary::get<I>(T::W_MetaObjectCreatorHelper::objectInfo.methods).func;
         using P = QtPrivate::FunctionPointer<std::remove_const_t<decltype(f)>>;
         auto _t = QtPrivate::ConnectionTypes<typename P::Arguments>::types();
-        uint arg = *reinterpret_cast<int*>(_a[1]);
+        uint arg = *reinterpret_cast<uint*>(_a[1]);
```

Comment:
isn't there a change of behaviour introduced here ?

Reply:
@jcelerier I don't see it.   Casting between int and uint should not be a problem in this case.